common/src/test/java/org/apache/rocketmq/common/protocol/body/KVTableTest.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.rocketmq.common.protocol.body;
```

Comment:
Please add the apache license header before the code.

Reply:
OK! Thank you!